databases/utils.py

```diff
@@ -245,5 +254,8 @@ class RawQueries(BaseModel):
             SET title = 'updated title'
             WHERE id = 'sdldsd'
         """,
+        select_tx_isolation="""
+            PRAGMA read_uncommitted = 1
```

Comment:
question: this query looks like it would mutate? Am I misunderstanding what this is doing?

Reply:
As far as I understand SQLite not support commands like `PRAGMA read_committed = 1` or `PRAGMA serializable = 1`. Only `PRAGMA read_uncommitted = 0` or `PRAGMA read_uncommitted = 1`.

Anyway execution of `PRAGMA` is [skipped](https://github.com/RobertCraigie/prisma-client-py/blob/110645e05695d62071245e4636d861bcaa15877a/databases/tests/test_transactions.py#L250) in tests.

Reply:
Forget the previous comment 🤦. SQLite doesn't have the way to query the current transaction isolation level.

I force pushed new version.